minifi/minifi-toolkit/minifi-toolkit-configuration/src/main/java/org/apache/nifi/minifi/toolkit/configuration/json/TransformNifiCommandFactory.java

```diff
@@ -63,8 +65,11 @@ private int transformNifiToJson(String[] args) {
             RegisteredFlowSnapshot registeredFlowSnapshot = readNifiFlow(sourceNiFiJsonPath);
             VersionedDataflow versionedDataflow = new VersionedDataflow();
             versionedDataflow.setRootGroup(registeredFlowSnapshot.getFlowContents());
-            versionedDataflow
-                    .setParameterContexts(new ArrayList<>(registeredFlowSnapshot.getParameterContexts().values()));
+            versionedDataflow.setParameterContexts(new ArrayList<>(registeredFlowSnapshot.getParameterContexts().values()));
+
+            ConfigSchema configSchema = new ConfigSchema(Collections.emptyMap());
+            versionedDataflow.setMaxTimerDrivenThreadCount(configSchema.getCoreProperties().getMaxConcurrentThreads().intValue());
```

Comment:
This will always result in the DEFAULT_MAX_CONCURRENT_THREADS, what about just simply reference it without instantiating ConfigSchema?
```
versionedDataflow.setMaxTimerDrivenThreadCount(CorePropertiesSchema.DEFAULT_MAX_CONCURRENT_THREADS);
```